security/firely-auth/firely-auth-releasenotes.rst

```diff
@@ -3,6 +3,25 @@
 Release notes
 =============
 
+.. _firelyauth_releasenotes_3.3.0:
+
+Release 3.3.0, March xx, 2024
+-----------------------------
+
+Security
+^^^^^^^^
+#. Disabling 2FA authentication for a client will now require a 2FA token from the user as an additional security step
```

Comment:
Note that we can still log in with a user without 2FA without issue, so it is not required by FA, but can be configured for the user. Unless this behavior s unwanted?

Reply:
Thanks for double checking, that’s indeed the intended behavior